build-logic/uber-plugins/src/main/kotlin/gradlebuild.kotlin-library.gradle.kts

```diff
@@ -76,6 +76,6 @@ fun KotlinCompile.configureKotlinCompilerForGradleBuild(launcher: Provider<JavaL
             "-Xskip-metadata-version-check"
         )
         jvmTarget = "1.8"
-        jdkHome = launcher.get().metadata.installationPath.asFile.absolutePath
+        kotlinJavaToolchain.toolchain.use(launcher)
```

Comment:
Does the Kotlin compiler automatically use the same toolchain as Java if we've set it there? we might be able to remove this entirely.

Reply:
Good point, @big-guy. I checked the docs at https://kotlinlang.org/docs/gradle.html#gradle-java-toolchains-support and that seems to be the case:

> You can set a toolchain via the java extension, and Kotlin compilation tasks will use it:

I'll remove it.